.env

```diff
@@ -3,7 +3,7 @@
 # Override any of these variables in `.env.local`
 # if the values are different on your machine
 
-ADMINISTRATOR_CONTACT_PHONE="206-615-0226"
+ADMINISTRATOR_CONTACT_PHONE=206-615-0226
```

Comment:
Small change - I noticed that the app was rendering the quotes in the UI. Turns out this file format treats quotes as literal characters.